webauthn.h

```diff
@@ -304,6 +305,9 @@ typedef struct _WEBAUTHN_CREDENTIAL_DETAILS {
 
     // User Info
     PWEBAUTHN_USER_ENTITY_INFORMATION   pUserInformation;
+
+    // Removable or not.
+    BOOL bRemovable;

```

Comment:
I think you need to uprev 
#define WEBAUTHN_GET_CREDENTIALS_OPTIONS_VERSION_2          2
#define WEBAUTHN_GET_CREDENTIALS_OPTIONS_CURRENT_VERSION    WEBAUTHN_GET_CREDENTIALS_OPTIONS_VERSION_2